Sources/Shared/VimeoClient.swift

```diff
@@ -108,17 +110,32 @@ final public class VimeoClient {
     /// - Parameters:
     ///   - appConfiguration: Your application's configuration
     ///   - configureSessionManagerBlock: a block to configure the session manager
-    convenience public init(appConfiguration: AppConfiguration, configureSessionManagerBlock: ConfigureSessionManagerBlock?) {
-        self.init(appConfiguration: appConfiguration, sessionManager: VimeoSessionManager.defaultSessionManager(appConfiguration: appConfiguration, configureSessionManagerBlock: configureSessionManagerBlock))
+    convenience public init(
+        appConfiguration: AppConfiguration,
+        configureSessionManagerBlock: ConfigureSessionManagerBlock? = nil,
+        reachabilityMonitor: ReachabilityManagingType? = VimeoReachabilityProvider.reachabilityManager
```

Comment:
Should we maybe switch the order of the parameters so trailing-closure syntax could be taken advantage of?